web_form_banner/models/ir_model.py

```diff
@@ -49,7 +54,7 @@ def fields_view_get(
                     "data-rule-id": str(rule.id),
                     "data-model": self._name,
                     "data-default-severity": (rule.severity or "danger"),
-                    "data-trigger-fields": ",".join(rule.trigger_field_ids.mapped("name")) if getattr(rule, "trigger_field_ids", False) else "",
+                    "data-trigger-fields": triggers_fields,
```

Comment:
```suggestion
                    "data-trigger-fields": ",".join(rule.trigger_field_ids.mapped("name"))
```
@AungKoKoLin1997 I think this should suffice.